### PR TITLE
formatting and clean up

### PR DESCRIPTION
**Pull request by TheAveryRodriguez**
## Description

Went through the app and did some small cleaning. Formatted our code to be uniform through and through.

## Type of change

- [ ] fix
- [ ] feat
- [ ] test
- [x] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

**Review by mbkuhl — CHANGES_REQUESTED** [review]
There's like two formatting rules that I think we disagree on, and unfortunately they're pretty prevalent in the code base. The first one is hash white space in a hash. I keep it in for readability, and I think that's also standard according to the rubycop docs.
Second is on the create/update blocks, I've been making all the attributes line up with each other as opposed to strictly obeying the single indent. This was also readability. I think that may also be standard in a lot of places, but I couldn't find any specific source on that one. I know a lot of our instructors format attributes that way.

I think everything I saw that wasn't those two things I likes

File: app/controllers/api/v0/rounds/votes_controller.rb [hunk 1]
```diff
@@ -8,19 +7,19 @@ def create
       render json: ErrorSerializer.new(ErrorMessage.new(exception.message, 404)).error_json, status: :not_found
     end
     if @round.open?
-      vote = Vote.find_by( user_id: params[:user_id],
-                           round_id: params[:round_id])
+      vote = Vote.find_by(user_id: params[:user_id],
+        round_id: params[:round_id])
       if vote
-        vote.update( lat: params[:lat],
```
Comment (mbkuhl) [inline app/controllers/api/v0/rounds/votes_controller.rb:14]: This is one example where I liked the previous version more than the change. Having the extra indents serves a purpose in saying "This whole block serves one purpose, and you can skip over the rest if you know what the first line does". Without the extra indent, it looks just like another method at a glance.

File: app/controllers/api/v0/users/games_controller.rb [hunk 2]
```diff
@@ -25,13 +25,13 @@ def invite_rsvp
     if user_game
       if params[:rsvp] == "accept"
         user_game.update(invitation: 1)
-        render json: { data: { message: "Successfully Accepted Invitation"}}, status: 201
```
Comment (mbkuhl) [inline app/controllers/api/v0/users/games_controller.rb:28]: This is another example of where I liked the previous version better than the original. Having white space allows your brain to be able to separate the values from their surroundings and process the data more quickly.

File: app/controllers/api/v0/cron_controller.rb
Comment (mbkuhl) [inline app/controllers/api/v0/cron_controller.rb:1]: I love the changes like this

File: app/controllers/api/v0/users/rounds_controller.rb
Comment (mbkuhl) [inline app/controllers/api/v0/users/rounds_controller.rb:1]: Again, love the changes like this

File: app/models/round.rb [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
+      self.close_date = find_close_date
     end
-    unless self.process_date
-      self.process_date = self.find_process_date
+    unless process_date
+      self.process_date = find_process_date
     end
-    self.game_type = self.game.game_type
+    self.game_type = game.game_type
   end
```
Comment (mbkuhl) [inline app/models/round.rb:10]: Love these changes too

File: app/controllers/api/v0/users_controller.rb [hunk 4]
```diff
@@ -19,22 +19,20 @@ def create
...
     @user = User.find_by(email: params[:email], verified: 2)
-    unless @user
-      @user = User.create(email: params[:email], 
-                          verified: 2,
-                          username: User.username_generator, 
-                          password: "AccThruOAuth1", 
-                          password_confirmation: "AccThruOAuth1", 
-                          verification_token: nil)
-    end
+    @user ||= User.create(email: params[:email],
```
Comment (mbkuhl) [inline app/controllers/api/v0/users_controller.rb:22]: Looks a lot better than the unless. (I still want the indents back on the lines below)

**Review by mbkuhl — COMMENTED** [review]
Comments in code base. Only did the first bit, because it looks like those comments apply to all the files

File: app/controllers/api/v0/rounds/votes_controller.rb (hunk 1 above)
Comment (TheAveryRodriguez) [reply app/controllers/api/v0/rounds/votes_controller.rb:14]: Currently updating to include greater indentation.

File: app/controllers/api/v0/users/games_controller.rb (hunk 2 above)
Comment (TheAveryRodriguez) [reply app/controllers/api/v0/users/games_controller.rb:28]: I am unsure of what lines you are referring to specifically.

**Review by TheAveryRodriguez — COMMENTED** [review]
Went through and updated the revisions. Only had 1 comment I think. Should be good to PR

**Comment by mbkuhl** [comment]
This was the single most successful merge conflict resolution ever.

**Pull request merged**